webapp/Connector/src/view/Grid.js

```diff
@@ -579,7 +618,7 @@ Ext.define('Connector.view.Grid', {
         var model = this.getModel(),
             modelMap = {},
             columns,
-            models = model.get('metadata').columnModel,
+            models = model.getActiveSheetMetadata().columnModel,
```

Comment:
Maybe `columnModels` would be a better variable name since `model` is present and is something else entirely.